App.js

```diff
@@ -1,19 +1,24 @@
 import { StatusBar } from 'expo-status-bar';
 import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import { StyleSheet, View } from 'react-native';
+import Amplify from 'aws-amplify'
+import awsconfig from './src/aws-exports'
+import Home from './src/Home';
+import Header from './src/Header';
 
+Amplify.configure(awsconfig)
```

Comment:
Nit: add a line break between this line and the `App` declaration

Reply:
Got it.